packages/components/src/search/autocompleters/coupons.js

```diff
@@ -29,7 +29,7 @@ export default {
 		let payload = '';
 		if ( search ) {
 			const query = {
-				search,
+				search_code: search,
```

Comment:
Can we avoid renaming this? _I sound like a broken record..._

Reply:
No problemo- done!